<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemStatisticProgression</class>
 <widget class="QDialog" name="DialogSystemStatisticProgression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistic progression...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0" colspan="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>508</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelRandom">
         <property name="text">
          <string>Random variation:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="PanelProgressionTable" name="panelProgressionTable" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>288</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QRadioButton" name="radioButtonFix">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Fix</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QRadioButton" name="radioButtonFormula">
         <property name="text">
          <string>Formula</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveValueRandom" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBoxInformation">
         <property name="title">
          <string>Information</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelStatisticID">
              <property name="text">
               <string>Statistic ID:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>118</width>
                <height>48</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxStatistic">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelMaximumValue">
              <property name="text">
               <string>Maximum value:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PanelPrimitiveValue" name="panelPrimitiveValueMax" native="true">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>27</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="PanelPrimitiveValue" name="panelPrimitiveValueFormula" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelProgressionTable</class>
   <extends>QWidget</extends>
   <header>panelprogressiontable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemStatisticProgression</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemStatisticProgression</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
